CHANGELOG_ANDROID.md

```diff
@@ -1,5 +1,20 @@
 # Changelog
 
+## [1.13.85](https://github.com/brave/brave-browser/releases/tag/v1.13.85)
+
+ - Implemented cross-platform promotion modal. ([#10571](https://github.com/brave/brave-browser/issues/10571))
+ - Implemented "Refresh Status" button on rewards panel to check for publisher status. ([#10005](https://github.com/brave/brave-browser/issues/10005))
```

Comment:
Can remove `for` from the sentence. Reads better as:

* `Implemented "Refresh Status" button on rewards panel to check publisher status.`